{{ cookiecutter.repo_name }}/README.md

```diff
@@ -88,7 +88,7 @@ An overview of the folder structure, and the top-level files can be found [here]
 
```

Comment:
Between Lines-11-18 (requirements), you should probably add a jinja if statement to include R as a requirement if the R hooks are being installed. See Lines 15-16 about how to do this!

Reply:
Should be done here, https://github.com/ukgovdatascience/govcookiecutter/pull/9/commits/387f42dfad8fbf0c3022445f86b97243091dab71.

Reply:
Yep minor issue, discussed and resolved in later [review](https://github.com/ukgovdatascience/govcookiecutter/pull/9#discussion_r518622435).